### ISSUE TITLE
Add missing alt texts to posters

### ISSUE THREAD

**Issue by sergei-kalugin** [MADStudioNU/the-pulter-project#774]
https://docs.google.com/spreadsheets/d/1X7zMzNxrA3aBwSAOqZytjyVSo2Gn4fDGwcth61fBkOo/edit#gid=0
